internal/pkg/scaffold/ansible/k8s_status.go

```diff
@@ -343,7 +343,19 @@ class KubernetesAnsibleStatusModule(KubernetesAnsibleModule):
             'changed': True
         }
 
+    #  clean_last_transition_time removes lastTransitionTime attribute from each status.conditions[*].
+    #  That is because object_contains function which is called after clean_last_transition_time, compares
+    #  if new conditions are subset of old conditions, but due to lastTransitionTime
+    #  they are never subset. New conditions should not contain lastTransitionTime too. It will be set by kubernetes.
+    def clean_last_transition_time(self, status):
+      if status.has_key("conditions"):
```

Comment:
You can merge the lines 351 and 352 by using old_conditions = status.get("conditions", [])